{"classifiers": ["Development Status :: 5 - Production/Stable", "Environment :: Web Environment", "Intended Audience :: Developers", "License :: OSI Approved :: Zope Public License", "Programming Language :: Python", "Programming Language :: Python :: 2", "Programming Language :: Python :: 2.7", "Programming Language :: Python :: 3", "Programming Language :: Python :: 3.3", "Programming Language :: Python :: 3.4", "Programming Language :: Python :: 3.5", "Programming Language :: Python :: 3.6", "Programming Language :: Python :: Implementation :: CPython", "Programming Language :: Python :: Implementation :: PyPy", "Natural Language :: English", "Operating System :: OS Independent", "Topic :: Internet :: WWW/HTTP", "Framework :: Zope3"], "extensions": {"python.details": {"contacts": [{"email": "zope-dev@zope.org", "name": "Zope Corporation and Contributors", "role": "author"}], "document_names": {"description": "DESCRIPTION.rst"}, "project_urls": {"Home": "http://github.com/zopefoundation/zope.errorview"}}}, "extras": ["browser", "test"], "generator": "bdist_wheel (0.29.0)", "keywords": ["zope3", "http", "error", "view"], "license": "ZPL 2.1", "metadata_version": "2.0", "name": "zope.errorview", "run_requires": [{"requires": ["setuptools", "zope.component", "zope.i18n", "zope.i18nmessageid", "zope.interface", "zope.publisher", "zope.security"]}, {"extra": "browser", "requires": ["zope.authentication", "zope.browser", "zope.browserpage"]}, {"extra": "test", "requires": ["zope.authentication", "zope.browser", "zope.browserpage", "zope.testing", "zope.testrunner"]}], "summary": "Basic HTTP and Browser exception views.", "test_requires": [{"requires": ["zope.authentication", "zope.browser", "zope.browserpage", "zope.testing", "zope.testrunner"]}], "version": "1.1"}